usaspending_api/database_scripts/job_archive/backfill_per_transaction_exec_comp.py

```diff
@@ -33,15 +33,15 @@
 BROKER_FABS_SELECT_SQL = """
 SELECT
     published_award_financial_assistance_id,
-    high_comp_officer1_full_na,
+    UPPER(high_comp_officer1_full_na),
```

Comment:
Is this suppose to be all upper case? Since it is dealing with names it might affect readability in cases where names have a mixture of upper and lower cases.

Reply:
I don't think it was the correct decision to uppercase all text fields, but this is the way it has been for perhaps 2 years